### DISCUSSION TITLE
Uploading vs IMAGES_FOLDER_OPTIONAL

### DISCUSSION THREAD

**Discussion by zylervega** [TheLastBen/fast-stable-diffusion#964] [Q&A]
(fast-DreamBooth.ipynb)

I have ~15,000 512x512 images.  I uploaded them to my gdrive, then specified the directory with IMAGES_FOLDER_OPTIONAL.

However when I ran the cell it began "uploading" them "to the instance".

How can I avoid having to "upload" twice?  Where are the files "uploading" to, considering they are already in my gdrive and available for the notebook to use?

**Comment by TheLastBen** [comment]
They are uploaded to the colab temporary storage where they are cropped.

**Reply by zylervega** [reply]
In my case I had unchecked the "crop" checkbox as I'd already cropped them.
